package.json

```
@@ -26,7 +26,7 @@
         "defaults"
     ],
     "scripts": {
-        "dev-server": "encore dev-server",
+        "dev-server": "encore dev-server --server-type https",
```

```suggestion
        "dev-server": "encore dev-server",
```

Kinda a slip-up, should be reverted as well as it'll be implemented in https://github.com/contao/contao/pull/8836

---

Ah these are merge issues.